### ISSUE TITLE
Error when importing burp xml

### ISSUE THREAD

**Issue by Techbrunch** [dradis/dradis-ce#359]
### Steps to reproduce

Export scan from import and import it in Dradis CE

### Expected behavior

No errors

### Actual behavior

```
[ActiveJob] [UploadJob] [0f64fd4b-0b1c-4e09-8a1b-d4b0e0313848] Error performing UploadJob (Job ID: 0f64fd4b-0b1c-4e09-8a1b-d4b0e0313848) from Async(dradis_upload) in 0.2ms: ArgumentError (unknown keyword: project_id):
/Users/xxx/Documents/github/dradis-ce/app/jobs/upload_job.rb:4:in `perform'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/execution.rb:37:in `block in perform_now'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:108:in `block in run_callbacks'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/i18n-1.0.1/lib/i18n.rb:284:in `with_locale'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/translation.rb:7:in `block (2 levels) in <module:Translation>'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:117:in `instance_exec'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:117:in `block in run_callbacks'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/logging.rb:24:in `block (4 levels) in <module:Logging>'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `block in instrument'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `instrument'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/logging.rb:23:in `block (3 levels) in <module:Logging>'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/logging.rb:44:in `block in tag_logger'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:69:in `block in tagged'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:26:in `tagged'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:69:in `tagged'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/logging.rb:44:in `tag_logger'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/logging.rb:20:in `block (2 levels) in <module:Logging>'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:117:in `instance_exec'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:117:in `block in run_callbacks'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:135:in `run_callbacks'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/execution.rb:33:in `perform_now'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/execution.rb:22:in `block in execute'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:108:in `block in run_callbacks'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/railtie.rb:26:in `block (4 levels) in <class:Railtie>'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/execution_wrapper.rb:85:in `wrap'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/reloader.rb:68:in `block in wrap'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/execution_wrapper.rb:85:in `wrap'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/reloader.rb:67:in `wrap'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/railtie.rb:25:in `block (3 levels) in <class:Railtie>'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:117:in `instance_exec'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:117:in `block in run_callbacks'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:135:in `run_callbacks'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/execution.rb:20:in `execute'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activejob-5.1.6/lib/active_job/queue_adapters/async_adapter.rb:68:in `perform'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/concurrent-ruby-1.0.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:348:in `run_task'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/concurrent-ruby-1.0.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:337:in `block (3 levels) in create_worker'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/concurrent-ruby-1.0.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:320:in `loop'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/concurrent-ruby-1.0.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:320:in `block (2 levels) in create_worker'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/concurrent-ruby-1.0.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:319:in `catch'
/Users/xxx/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/concurrent-ruby-1.0.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:319:in `block in create_worker'
```

### System configuration
**Dradis version**:

3.9

**Ruby version**:

ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin17]

**OS version**:

MacOS HighSierra

**Comment by aapomm** [comment]
Hi @Techbrunch. Can you confirm that this has been resolved in CE v3.10?

**Comment by etdsoft** [comment]
Closing due to inactivity, re-open if needed.
